databroker/broker.py

```diff
@@ -510,7 +599,7 @@ def fill_event(self, event, inplace=True,
     def get_events(self, headers, fields=None, stream_name=ALL, fill=False,
```

Comment:
This function now very problematic.   It should probably take exactly 1 header and switch to a single stream.